modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -356,6 +368,81 @@ private void configureIndexer() {
         checkSuccessfulResponse(response, "Could not configure the indexing service");
     }
 
+    /**
+     * Loads the sample buckets through the REST API.
+     */
+    private void loadSampleBuckets() {
+        if (sampleBuckets.isEmpty()) {
+            logger().debug("No sample buckets to load, skipping.");
+            return;
+        }
+        logger().debug("Loading sample buckets: " + sampleBuckets);
```

Comment:
Use property placeholder instead of string concatenation

Reply:
fixed